sdk/appconfiguration/feature-management/feature/management/_defaultfilters.py

```diff
@@ -0,0 +1,112 @@
+# ------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+from ._featurefilters import FeatureFilter
+from datetime import datetime, timezone
+from email.utils import parsedate_to_datetime
+import logging
+import hashlib
+
+
+class TargetingException(Exception):
+    pass
+
+
+class TimeWindowFilter(FeatureFilter):
+    def evaluate(self, context, **kwargs):
+        """Determain if the feature flag is enabled for the given context"""
+        start = context.get("parameters", {}).get("Start")
+        end = context.get("parameters", {}).get("End")
+
+        current_time = datetime.now(timezone.utc)
+
+        if not start and not end:
+            logging.warn("TimeWindowFilter: Either Start or End are required to have as a parameter")
```

Comment:
Instead of hardcoding "TimeWindowFilter", can we get the class name? Its the same as C# `nameof` expression and it would help to keep the error message always in sync with class name.